server/src/main/resources/nacos-config.txt

```diff
@@ -42,4 +42,5 @@ recovery.asyn-committing-retry-delay = 5
 recovery.rollbacking-retry-delay = 5
 recovery.timeout-retry-delay = 5
 transaction.undo.data.validation=true
```

Comment:
Spaces on both sides of =

Reply:
Maybe using space is wrong in this file.  Do I need to remove the space?

```
store.db.min-conn=1
store.db.max-conn=3
store.db.global.table=global_table
store.db.branch.table=branch_table
store.db.query-limit=100
lock.mode=memory
lock.db.lock-table=lock_table
recovery.committing-retry-delay = 5    <--
recovery.asyn-committing-retry-delay = 5   <--
recovery.rollbacking-retry-delay = 5   <--
recovery.timeout-retry-delay = 5   <--
transaction.undo.data.validation=true
```